GoogleSignIn/Sources/GIDGoogleUser.m

```diff
@@ -106,8 +113,31 @@ - (GIDConfiguration *)configuration {
   return _cachedConfiguration;
 }
 
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
+- (id<GTMFetcherAuthorizationProtocol>)fetcherAuthorizer {
```

Comment:
Rather than creating a new instance each time the property is accessed, let's create an instance at initialization and return a reference to that instance.  This could be done by dropping the getter and simply setting the property ivar.

Reply:
What's preventing us from doing this now?  Implementing `OIDAuthStateChangeDelegate` doesn't seem like a requirement as we're not currently observing changes that may originate from the multiple instances of `GTMAppAuthFetcherAuthorization` that can be generated.

Reply:
Done.